.github/workflows/release-discover-branches.yaml

```diff
@@ -39,6 +39,10 @@ jobs:
         working-directory: ./src/github.com/openshift-knative/hack
         run: make discover-branches ARGS="--config config/"
 
+      - name: Generate Action
+        working-directory: ./src/github.com/openshift-knative/hack
+        run: make generate-action
```

Comment:
Could we rename this action (and the Makefile target) to something like "Generate GH Actions for Konflux Manifests"? (I am not good in naming, but I have to look up often, what this `generate-action` does :( )

Reply:
But not super related to this PR though

Reply:
`generate-konflux-action` ?

Reply:
it's also doing openshift/release CI updates

Reply:
generate-ci-action

The actual displayed name is `Generate CI config`

Reply:
Renamed to `generate-ci-action`